cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraStorageSettings.java

```diff
@@ -63,6 +63,7 @@ public class CassandraStorageSettings extends CoreStorageSettings {
     //when cassandra driver tries to convert Instant.MAX to milliseconds using toEpochMilli() it causes long overflow.
     public static final Instant MAX_EPOCH_INSTANT = Instant.ofEpochMilli(Long.MAX_VALUE);
     public static final Instant DEFAULT_PAGE_REMOVE_TIME = MAX_EPOCH_INSTANT;
+    public static final long DEFAULT_INIT_OPERATORS_DURATION_MILLIS = 10_000;

```

Comment:
question: do we believe that we need milliseconds resolution here? I don't think it can be that fast (considering the fact that the default timeout is 10 seconds). Maybe we should use seconds instead of milliseconds?